lib/ansible/modules/apt.py

```diff
@@ -1257,7 +1257,7 @@ def main():
         #    made any more complex than it already is to try and cover more, eg, custom interpreters taking over
         #    system locations)
 
-        apt_pkg_name = 'python3-apt' if PY3 else 'python-apt'
+        apt_pkg_name = 'python3-apt'
```

Comment:
The interpreters list should exclude '/usr/bin/python2' too.

Reply:
Good catch, done.